lib/kafka_ex.ex

```diff
@@ -29,10 +31,10 @@ defmodule KafkaEx do
   def create_worker(name, worker_init \\ [])
 
   def create_worker(name, worker_init) do
-    worker_init = case worker_init do
-      [] -> [uris: Application.get_env(:kafka_ex, :brokers)]
-      _   -> worker_init
-    end
+    defaults = [uris: Application.get_env(:kafka_ex, :brokers),
+                consumer_group: Application.get_env(:kafka_ex, :consumer_group)]
+
+    worker_init = Keyword.merge(defaults, worker_init)
```

Comment:
I expect that if I call `create_worker(:name, [keyword list])`, it will merge the keyword list with defaults (the uris and consumer group set in the config).  The original version does not work that way at all.  With the original version, if I call `create_worker(:name, consumer_group: "foo")` or `create_worker(:name, [consumer_group: "foo"])` (they are equivalent), the worker will fail to start because the uris will not be populated.  I would have to call `create_worker(:name, consumer_group: "foo", uris: [....])`, which seems odd if I want to use the uris that are already present in my App config.

I thought `Keyword.merge/2` was fair game here, since the function accepts a `Keyword.t` as its argument.  If the function name isn't clear enough, I could add a comment.

I don't understand your comment about performance at all.  What are your performance concerns with this?